data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/teleport/no_teleport.cfg

```diff
@@ -0,0 +1,52 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [teleport]
+##
+# Actions:
+# Move side 1's leader to the edge of the map
+# Place a village next to side 1's leader and side 2's leader
```

Comment:
In both docs and code, this needs to assign both villages to side 1; both for the usual definition of teleport, and because the test would currently pass if moving onto 1,2 used all of the unit's movement to capture the village.

Reply:
Yeah, I was surprised too. I don't think there's anything to update for this comment then though?

Reply:
Agreed, no change needed for this comment.